package-res/resources/web/pentaho/visual/base/events/DidSelect.js

```diff
@@ -26,10 +26,10 @@ define([
    * @description This event is triggered when
    * the {@link pentaho.visual.base.Model#selectAction|Select Action} flow ends without any failures.
    *
-   * @extends pentaho.visual.base.events.Did
+   * @extends pentaho.lang.Event
    * @event "did:select"
```

Comment:
:white_check_mark: ( removed the @event tag for now, but we must discuss this at some point)